scipy/sparse/__init__.py

```diff
@@ -48,6 +48,14 @@
    rand - Random values in a given shape
    random - Random values in a given shape
 
+Save and load sparse matrices:
+
+.. autosummary::
+   :toctree: generated/
+
+   save - Save a sparse matrix
+   load - Load a sparse matrix
```

Comment:
I agree with @pv that `read_npz`/`load_npz` might be a better API, especially if we plan on adding more kinds of formats to the new `matrix_io` module.